elements/rh-blockquote/docs/10-style.md

```diff
@@ -1,100 +1,200 @@
-A default blockquote consists of three required elements, a quote icon, text, 
-and attribution. It can be left- or center-aligned on large and small screens. A 
-left-aligned blockquote can also feature additional elements, like large text 
-styles and components on top or to the right whereas a center-aligned blockquote 
-can't include anything else.
+## Style
+A blockquote is a combination of elements used to give visual prominence 
+to a quotation. By default, a blockquote includes a quote icon, quotation 
+text, and citation text <strong>at a minimum</strong>. A blockquote may 
+also include the following optional elements:
+ - Emphasis border
+ - Logo
+ - Title and heading text
+ - Interactive elements or layouts like a video or card
+
+
+![Anatomy image of a blockquote with numbered annotations]({{ '../blockquote-anatomy.png' | url }}) {style="margin-block:var(--rh-space-2xl);--inline-img-max-width:653px;"}
+1. Quote icon
+2. Quotation text
+3. Citation text
+4. Logo or text placeholder
+5. Emphasis border
+
+## Sizes
+
+![Image of two blockquotes, default size on the left and large size on the right]({{ '../blockquote-style-sizes.png' | url }}) {style="margin-block:var(--rh-space-2xl);--inline-img-max-width:872px;"}
+
+| Size              | Element                 | Current value  |
+|-------------------|-------------------------|----------------|
+| Default           | Text size - quotation   | 20px, 1.25rem  |
+| Default           | Line height - quotation | 30 (1.5)       |
+| Large             | Text size - quotation   | 28px, 1.75rem  |
+| Large             | Line height - quotation | 36.4 (1.3)     |
+| Default and Large | Text size - citation    | 14px, 0.875rem |
+| Default and Large | Line height - citation  | 21 (1.5)       |
+
+{.full-width}
```

Comment:
You'll want to update your branch and pull in the new tokensTable shortcode

```Suggestion
{% tokensTable %}
| Size              | Element                 | Current value  |
|-------------------|-------------------------|----------------|
| Default           | Text size - quotation   | 20px, 1.25rem  |
| Default           | Line height - quotation | 30 (1.5)       |
| Large             | Text size - quotation   | 28px, 1.75rem  |
| Large             | Line height - quotation | 36.4 (1.3)     |
| Default and Large | Text size - citation    | 14px, 0.875rem |
| Default and Large | Line height - citation  | 21 (1.5)       |

{.full-width}

{% endtokensTable %}
```

Note this will hide these tables, this is expected right now